lms/static/js/edxnotes/views/tabs/tags.js

```diff
@@ -0,0 +1,129 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'gettext', 'jquery', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
+    'js/edxnotes/views/tab_view'
+], function (gettext, $, _, NoteGroupView, TabPanelView, TabView) {
+
+    var TagsView = TabView.extend({
+        scrollToTag: function(tagName) {
+            var titleElement, displayedTitle;
+            if (!this.tabModel.isActive()) {
+                this.tabModel.activate();
+            }
+
+            displayedTitle = this.contentView.titleMap[tagName.toLowerCase()];
+            titleElement = this.$el.find('.tags-title').filter(
+                function(){ return $(this).text() === displayedTitle; }
+            );
+            $('html,body').animate({
+                scrollTop: titleElement.offset().top - 10
+            },'slow');
+        },
+
+        initialize: function (options) {
+            TabView.prototype.initialize.call(this, options);
+            _.bindAll(this, 'scrollToTag');
+            this.options.scrollToTag = this.scrollToTag;
+        },
+
+        PanelConstructor: TabPanelView.extend({
+            id: 'tags-panel',
+            title: 'Tags',
+            noTags: gettext('[no tags]'),  // User-defined tags cannot have spaces, so no risk of a collision.
```

Comment:
I would add a translators comment here. I have no idea what "no tags" means, also you should clarify that the text should be translated (translators may not be sure if text within `[]` is a variable or not)